src/components/overlay/readme.md

```diff
@@ -0,0 +1,116 @@
+The `sbb-overlay` component provides a way to present content on top of the app's content and it's similar to the [sbb-dialog](/docs/components-sbb-dialog--docs).
+It offers the following features:
+
+- disables scrolling of the page content while open;
+- manages focus properly by setting it on the first focusable element;
+- has a close button, which is always visible;
+- can display a back button;
+- adds the appropriate ARIA roles automatically.
+
+```html
+<sbb-overlay id="my-overlay">
+  <p>Overlay content.</p>
+</sbb-overlay>
+```
+
+## Slots
+
+There is only one unnamed slot to provide the overlay content.
+
+## Style
+
+Setting the `expanded` property will cause the `sbb-overlay` component to take up the full width of the page.
+
+It's possible to display the component in `negative` variant using the self-named property.
+
+```html
+<sbb-overlay negative>
+  <p>Overlay content.</p>
+</sbb-overlay>
+```
+
+The default `z-index` of the component is set to `1000`; to specify a custom stack order, the
```

Comment:
I remember we are removing this kind of information due to the new `CSS properties` table

Reply:
Good catch, just see other cases for the same where it is already migrated :)